### PR TITLE
Challenge2 day3

### PR DESCRIPTION
**Pull request by AineKiraboMbabazi**
**Sorting:**
Given a list containing integers, floats and one character strings, write a function that takes
a list and returns a dictionary with keys evens, odds, and chars. The value for evens is a list of sorted even numbers, the value for odds is a list of sorted odd numbers and chars is a list of sorted character strings.
list_sort([2,0,6,5,1,7,’z’,’a’])
returns {‘evens’: [0,2,6], odds: [1,5,7], chars:[‘a’,’z’]}
**Extra challenge**
Write a python program to find missing numbers from a list. Follow a test driven approach while completing this challenge
**Example**
Input : [ 1, 2, 3, 5, 6, 7,9]
Output: [4,8]

### CODE REVIEW

**Comment by AineKiraboMbabazi** [comment]
@dennisja

File: list_sorting.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            characters.append(i)
+            mydict['chars']=sorted(characters)
+
+        elif isinstance(i, int):
+            if i%2==0:
+                even.append(i)
+                mydict['evens']=sorted(even)
+            else:
+                odd.append(i)
+                mydict['odds']=sorted(odd)
```
Comment (dennisja) [inline list_sorting.py]: Remove sorting logic here
DO it out of the loop

File: list_sorting.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    
+    for i in list1:
+        if isinstance(i, str):
+            characters.append(i)
+            mydict['chars']=sorted(characters)
+
+        elif isinstance(i, int):
+            if i%2==0:
+                even.append(i)
+                mydict['evens']=sorted(even)
```
Comment (dennisja) [inline list_sorting.py]: Remove sorting logic here
DO it out of the loop

File: missingnumber.py [hunk 3]
```diff
@@ -0,0 +1,20 @@
+def missing_number_list(list1):
```
Comment (dennisja) [inline missingnumber.py]: Make the function cater for any numbers in a list. Even those greater than 10

**Pull request merged**